ui/app/components/project-input-variables/list-item.hbs

```diff
@@ -0,0 +1,116 @@
+<li data-test-input-variables-list-item>
+  {{#if (or this.isEditing this.isCreating)}}
+    <form class="pds-form" data-test-input-variables-form {{on "submit" this.saveVariable}}>
+      <div part="fields" class="card">
+        <div class="card-header">
+          <h4>{{t "form.project_variables_settings.title"}}</h4>
+        </div>
+        <fieldset class="pds-formFieldSet">
+          <div class="pds-formField">
+            <label for="input-variable-name" class="pds-fieldName">
+              {{t "form.project_variables_settings.variable_name"}}
+            </label>
+            <Pds::Input
+              @type="text"
+              data-test-input-variables-var-name
+              id="input-variable-name"
+              placeholder={{t "form.project_variables_settings.variable_name_placeholder"}}
+              value={{this.variable.name}}
+              {{on "input" (pick "target.value" (set this "variable.name"))}}
+            />
```

Comment:
Echoing @almonk’s suggestion, it’d be cool if this field auto-focused when it appears.

Reply:
so I have been pondering this one: 
- when creating, yes
- when editing, variable value changes would be more frequent than key changes, so maybe the second input should be focused? (but then we have inconsistency and that's weird)

Reply:
Ah yeah, I think you’re right. Focusing the value is the more likely case. Maybe we focus the `key` field if it’s empty? Otherwise the value?

Reply:
@almonk we should talk about this one. we can solve it in a separate PR.

Reply:
👍 to solving in a separate PR